cfp/models.py

```diff
@@ -12,6 +17,20 @@ class Proposal(models.Model):
          (6, 'Performant Python'),
          (7, 'Scalable Python'),
          (8, '/etc')])
-
+    speaker_bio = models.TextField(verbose_name="Speaker Bio", max_length=500)
```

Comment:
@mzdaniel okay. that make sense :) I talked to Daniel P today and he will create a new branch from my repo and work on the cfp forms. He will probably create a new one from scratch. I will keep working on the front-end.